subworkflows/local/bgc.nf

```diff
@@ -71,52 +70,24 @@ workflow BGC {
 
         }
 
-        if ( params.annotation_tool == 'prodigal' || params.annotation_tool == "pyrodigal" ) {
+        ch_antismash_input = gbks
+                                .filter {
+                                    meta, files ->
+                                        if ( meta.longest_contig < params.bgc_antismash_sampleminlength ) log.warn "[nf-core/funcscan] Sample does not have any contig reaching min. length threshold of --bgc_antismash_sampleminlength ${params.bgc_antismash_sampleminlength}. AntiSMASH will not be run for sample: ${meta.id}."
```

Comment:
@Darcy220606 filtering only happens for antismash, even if stats generated for all samples regardless of workflow activated, I was corret! :grimacing: